depthimage_to_navscan_rgbd/src/depthimage_to_navscan_rgbd.cpp

```diff
@@ -148,5 +156,6 @@ int main(int argc, char** argv)
         }
 
         pointcloud2_publisher.publish(pointcloud2_msg);
+        r.sleep();
```

Comment:
I did see that one. I think we should not put a sleep there. (Or a much smaller sleep.) Because not getting an image would result in an inconsistent rate.
```
0 start loop
0.1 done processing
1 start loop2
1 not getting image
2 start loop3
2.1 done processing
.
.
.
```
vs
```
0 start loop
0 getting image
0.1 done processing
1 start loop2
1 not getting image
1.01 not getting image
1.02 not getting image
1.03 getting image
1.13 done processing
2 start loop3
2.1 done processing
.
.
.
```

Reply:
My final suggestion is to use a sleep which is 10-100 times faster than the normal rate.

Reply:
I think it better to limit this process. just to prevent using too much resources.